docs/examples/101/basic-batch-account/main.bicep

```diff
@@ -0,0 +1,20 @@
+param BatchaccountName string

```

Comment:
can you use consistent camel-casing for the two parameters? So change `Batchaccountname` => `batchAccountName`

Reply:
@alex-frankel , Updated the naming as per nomenclature. Thank you.